### DISCUSSION TITLE
Error when running vcpkg install

### DISCUSSION THREAD

**Discussion by bagelbagelbeagle** [ChampSim/ChampSim#517] [Q&A] [answered]
Hi all,

I'm working on getting ChampSim built and running, but I keep hitting an error when I run `vcpkg/vcpkg install`. In particular, I see the following error:
```
error: building bzip2:x64-linux failed with: BUILD_FAILED
Elapsed time to handle bzip2:x64-linux: 4.8 s
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
```

If I continue ahead and try making ChampSim, I get the following error:
```
fatal error: fmt/core.h: No such file or directory
   24 | #include <fmt/core.h>
```
So, I think I'm missing some necessary dependencies that vcpkg should have downloaded. The shortened commit SHA for my ChampSim repo is aa336e83. I am running ChampSim in Ubuntu with WSL.

I've been looking for fix for this, but haven't found anything yet. I was wondering if anyone on here has encountered this error. Thanks!

**Comment by ngober** [comment]
I wonder if this has something to do with WSL. Did the failure come with any other information? Did you try updating vcpkg as it suggests?

**Comment by maccoymerrell** [comment, marked as answer]
I have seen errors like this before with WSL.

See if this fixes it:
sudo apt install pkg-config

Typically this happens because certain packages are just not default in wsl. Make sure its not complaining about a missing package before the bzip error.

**Reply by bagelbagelbeagle** [reply]
Many thanks! This fixed it
